README.md

```diff
@@ -189,19 +195,19 @@ An easy example would be to subscribe to changes of the master store, since that
 class Connection extends Component {
```

Comment:
Why not simply present how to use the decorator you've built? This inheritance is scary and heavy haha